packages/next/build/index.ts

```diff
@@ -154,11 +157,14 @@ export default async function build(dir: string, conf = null): Promise<void> {
   let hasPublicDir = false
 
   telemetry.record(
-    eventVersion({
+    eventVersion(userConfig || {}, {
       cliCommand: 'build',
       isSrcDir: path.relative(dir, pagesDir!).startsWith('src'),
       hasNowJson: !!(await findUp('now.json', { cwd: dir })),
       isCustomServer: null,
+      hasTypeScript: await hasTypeScript(pagesDir),
```

Comment:
This method is potentially super slow. We should be checking for presence of the following file:
```js
path.join(dir, 'tsconfig.json')
```